pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.github.devotedmc</groupId>
     <artifactId>hiddenore</artifactId>
-    <version>1.5.4</version>
+    <version>1.5.5</version>
```

Comment:
Convention in play here is MC version update, increment middle number -- so this would be 1.6.0